website/common/locales/en/inventory.json

```diff
@@ -1,7 +1,6 @@
 {
   "noItemsAvailableForType": "You have no <%= type %>.",
   "foodItemType": "Pet Food",
-  "eggsItemType": "Eggs",
```

Comment:
It does look like this one is used. You just don't find it on a `grep` because it's assembled dynamically in a template, for the Market drawer:

![image](https://user-images.githubusercontent.com/4501321/140419243-4539b851-5a90-47bb-89f2-653d2ad69bd9.png)

Reply:
Restored!